### ISSUE TITLE
Arg --date DATE seems not working. Eventually arg "date" is never passed to class Phockup

### ISSUE THREAD

**Issue by aty000** [ivandokov/phockup#96]
From phockup.py (main)

    return Phockup(
        args.input_dir,
        args.output_dir,
        dir_format=os.path.sep.join(DEFAULT_DIR_FORMAT),
        move=args.move,
        link=args.link,
        date_regex=args.regex,
        original_filenames=args.original_names,
        timestamp=args.timestamp,
        date_field=args.date_field,
        dry_run=args.dry_run,
    )

**Comment by jocmp** [comment]
I ran into this as well. The date argument still works in older versions. This is what I tried

```bash
sudo apt-get install python3 libimage-exiftool-perl -y
curl -L https://github.com/ivandokov/phockup/archive/1.5.11.tar.gz -o phockup.tar.gz
tar -zxf phockup.tar.gz
sudo mv phockup-* /opt/phockup
sudo ln -s /opt/phockup/phockup.py /usr/local/bin/phockup
```

v1.5.11 was the last update before this week.

Then run the usual

```bash
phockup <src_dir> <out_dir> -d YYYY-MM-DD
```

**Comment by ivandokov** [comment]
That's interesting. When I saw this issue I thought its coming from #87 but the diff does not show any issues with `date_field` https://github.com/ivandokov/phockup/commit/ac89f2dd822e8ca5186b999260d0e3fe14d7f6ee#diff-1e4466b25e08669b8bcdb6ae4a80550449c09003bd5f2074582b52f1e9bff34bR160-R171

I'll need to investigate it.

**Comment by ivandokov** [comment]
Actually the issue is there. I fixed it and just tagged version `1.5.24` with this fix.

**Comment by aty000** [comment]
Thank your very much for fixing it. I am not coding in phyton and I am too old anyway :-) From a cosmetic point of view I think that `"DEFAULT_DIR_FORMAT = ['%Y', '%m', '%d']"`, which is not used anymore in the phockup.py main module, could be moved and used in src/phockup.py. If it makes sense also for you, it should be possible to change in class Phockup() from :

`self.dir_format = args.get('dir_format', os.path.sep.join(['%Y', '%m', '%d']))` into 
`self.dir_format = args.get('dir_format', os.path.sep.join(DEFAULT_DIR_FORMAT))`

I thank you for the gr8 work on this tool which helped to re-org tons of family photos.

**Out of context of issue #96:** As you probably already know and if not, just for your info, the installation via "snap" completed successfully but snaps in general do not work well with remote resources. For example on a Ubuntu 20.04 with remote nfs mounts from a NAS, phockup - but also for other tools installed via snap -, is possible to get write access to nfs mounted disks because of snap limitations. Installing phockup in the manual way as you very well decribed in your github readme.md page - Linux (without snap) - , it worked then perfectly.
